Documentation.md

```diff
@@ -84,76 +94,76 @@ $ npm install -g gulp
 $ git clone https://github.com/aws/amazon-connect-streams
 $ cd amazon-connect-streams
 $ npm install
-$ npm run release 
+$ npm run release
 ```
 
-Find build artifacts in **release** directory -  This will generate a file called `connect-streams.js` and the minified version of the same `connect-streams-min.js`  - this is the full Connect Streams API which you will want to include in your page.
+Find build artifacts in **release** directory - This will generate a file called `connect-streams.js` and the minified version of the same `connect-streams-min.js` - this is the full Connect Streams API which you will want to include in your page.
 
 To run unit tests:
 ```
 $ npm run test
 ```
 
-### Using the AWS SDK and Streams
+## Using the AWS SDK and Streams
 Streams has a "baked-in" version of the AWS-SDK in the `./src/aws-client.js` file. Make sure that you import Streams before the AWS SDK so that the `AWS` object bound to the `Window` is the object from your manually included SDK, and not from Streams.
 
 ## Initialization
 Initializing the Streams API is the first step to verify that you have
 everything set up correctly and that you are able to listen for events.
 
 ### `connect.core.initCCP()`
-```
+```html
 <!DOCTYPE html>
 <meta charset="UTF-8">
 <html>
-	<head>
```

Comment:
Tabs replaced with spaces by auto formatting (since majority of document was using spaces).